common/runtime/src/main/java/io/quarkiverse/googlecloudservices/common/GcpCredentialProducer.java

```diff
@@ -25,6 +31,8 @@ public GoogleCredentials googleCredential() throws IOException {
             try (FileInputStream is = new FileInputStream(gcpConfiguration.serviceAccountLocation.get())) {
                 return GoogleCredentials.fromStream(is);
             }
+        } else if (tokenCredential.getToken() != null) {
+        	return GoogleCredentials.create(new AccessToken(tokenCredential.getToken(), null));
```

Comment:
This `null` is instead of the `Date` which apparently Google API can use to auto-refresh the token but we don't need, `quarkus-oidc` can handle it itself for the code flow ID token which also auto-refreshes the access token